README.md

```diff
@@ -1,37 +1,25 @@
 # strong-error-handler
 
-Error handler for use in development (debug) and production environments.
+This package is an error handler for use in both development (debug) and production environments.
 
-## Production mode
+In production mode, `strong-error-handler` omits details from error responses to prevent leaking sensitive information:
 
-- error responses are purposely left without detail in order to prevent leaking sensitive information.
 - For 5xx errors, the output contains only the status code and the status name from the HTTP specification.
 - For 4xx errors, the output contains the full error message (`error.message`) and the contents of the `details`
-  property (`error.details`). The latter is typically used by `ValidationError` to provide machine-readable details
-  about the validation problems.
+  property (`error.details`) that `ValidationError` typically uses to provide machine-readable details
+  about validation problems.
 
-## Debug Mode
+In debug mode, `strong-error-handler` returns full error stack traces and internal details of any error objects to the client in the HTTP responses.
 
-- Full error stack traces and internal details of any object passed as the error will be sent back to the client
-  in the HTTP responses when an error occurs.
-
-## Response Format
-
-- `strong-error-handler` currently supports HTML and JSON responses.
-- When the object is a standard Error object, the string provided by the stack property will be returned in HTML/text
-  responses.
-- When the object is a non-Error object, the result of `util.inspect` will be returned in HTML/text responses.
-- For JSON responses, the result will be an object with all enumerable properties from the object in the response.
-
-## Install
+## Installation
 
 ```bash
-$ npm install strong-error-handler
+$ npm install --save strong-error-handler
 ```
 
-## Usage
+## Use
```

Comment:
I am well aware that lots of documentation uses "usage" in this manner, but I _always_ prefer "use". [1] "Usage" is one of those favorite words of engineers and programmers because it sounds more sophisticated.  In this context an their noun form, the two words generally mean the same thing, and so I prefer the shorter and more  common word as a matter of principle.   I'm sure there are many instances of "usage" in our docs, but I have and will continue to change it when I run across it.  

So, in fact, our doc convention is "use" in this context.  However, if you want to keep "usage" here, I'm not going to insist.  Our doc style is and will continue to be "use", so at some point  this will be the outlier.

[1] http://blogs.transparent.com/english/use-versus-usage/
> Some people use the word “usage” as though it were just a fancier form of the word “use,” but this is not the best way to use the word.  In general, if either “use” or “usage” seems like it could work in a sentence the best choice is probably “use” as this is a more commonly used word ...